charts/catalog-api/values.yaml

```
@@ -291,6 +307,12 @@ importer:
     password: ''
     schemaSwitchEnabled: true
     schemaExtensions: extensions
+    tls:
+      enabled: false
+      rootCert: ''
+      cert: ''
+      key: ''
+      mode: ''
```

Какие возможные значения для mode ? Такие же как для [навигации](https://docs.2gis.com/ru/on-premise/deployment/navigation) ?
Давай так укажем по-дефолту?

```suggestion
      mode: verify-full	
```

---

указал